dotcom-rendering/src/components/Discussion.tsx

```diff
@@ -99,6 +99,9 @@ type State = {
 	hashCommentId: number | undefined;
 	totalPages: number;
 	loading: boolean;
+	topForm: { isActive: boolean };
+	replyForm: { isActive: boolean };
+	bottomForm: { isActive: boolean };
```

Comment:
For simplicity we used three forms: top, reply and bottom. We know this is the max number of forms we can have open in a discussion.